### ISSUE TITLE
Formatting bug: empty column headings

### ISSUE THREAD

**Issue by rauschma** [fcrespo82/vscode-markdown-table-formatter#15]
Before:

```
|                 |  |
|-----------------|---------|
| abc | defghi        |
|                 |         |
|                 |         |
```

After:

```
|-------------------|-----------|
| ----------------- | --------- |
| abc               | defghi    |
|                   |           |
|                   |           |
|                   |           ||
```

As soon as I insert a non-space in the first line, the table is formatted properly.

**Comment by fcrespo82** [comment]
Was able to recreate that bug.
Will look into it. Thanks for reporting.

**Comment by fcrespo82** [comment]
This should be fixed in 2.0.2
